.drone.yml

```diff
@@ -2018,8 +2018,8 @@ clone:
 environment:
   NETWORK: mainnet
   FROM_BLOCK_HEADER: 0
-  TO_BLOCK_HEADER: 130 # also check parameter --stop=
-  TO_BLOCK_HEADER_FOR_RPC: 3
+  TO_BLOCK_HEADER: 1000 # also check parameter --stop=
```

Comment:
@mambisi 
here 1000 -> 10000

and here `--stop=131` -> `--stop=10001`
https://github.com/simplestaking/tezedge/pull/503/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R2143

https://github.com/simplestaking/tezedge/pull/503/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R2161